docs/storage.md

```diff
@@ -64,7 +64,7 @@ If your local storage becomes corrupted for whatever reason, your best bet is to
 
 ## Remote storage integrations
 
-As outlined above, Prometheus's local storage is limited in its scalability and durability. Instead of trying to solve long-term storage in Prometheus itself, Prometheus has a set of interfaces that allow integrating with remote long-term storage systems.
+Prometheus's local storage is limited by single nodes in its scalability and durability. Instead of trying to solve clustered storage in Prometheus itself, Prometheus has a set of interfaces that allow integrating with remote clustered storage systems.
```

Comment:
Is it always clustered though? One of the targets is Graphite for example...?

Reply:
It's not always clustered, but the intended use case of remote storage is for larger / clustered storage systems.  The main point of this PR is to change the message about remote write being used for "long-term" storage, as the 2.0 TSDB is perfectly capable of handling long retention times.

Users are too often trying to implement remote write to TSDBs, like Graphite, thinking that they need to because Prometheus is unreliable.  All they really need to do is adjust the storage retention flag.

Single node Graphite on whisper is, IMO a worse solution.  However there are several clustered Graphite options that would be more suitable.